base/gfspapp/app.go

```diff
@@ -33,18 +33,19 @@ type GfSpBaseApp struct {
 	rcmgr      corercmgr.ResourceManager
 	chain      consensus.Consensus
 
-	approver   module.Approver
-	authorizer module.Authorizer
-	downloader module.Downloader
-	executor   module.TaskExecutor
-	gater      module.Modular
-	manager    module.Manager
-	p2p        module.P2P
-	receiver   module.Receiver
-	signer     module.Signer
-	uploader   module.Uploader
-	metrics    module.Modular
-	pprof      module.Modular
+	approver    module.Approver
+	authorizer  module.Authorizer
+	downloader  module.Downloader
+	executor    module.TaskExecutor
+	gater       module.Modular
+	manager     module.Manager
+	p2p         module.P2P
+	receiver    module.Receiver
+	signer      module.Signer
+	uploader    module.Uploader
+	metrics     module.Modular
+	pprof       module.Modular
+	blocksyncer module.Modular
```

Comment:
no other module need to interact with the blocksyncer?
if so, it can not be put in app struct, the main file have register the blocksyncer module is engouth.